gobblin-hive-registration/src/main/java/gobblin/hive/metastore/HiveMetaStoreBasedRegister.java

```diff
@@ -258,7 +260,9 @@ public void dropTableIfExists(String dbName, String tableName) throws IOExceptio
     try (AutoReturnableObject<IMetaStoreClient> client = this.clientPool.getClient()) {
       if (client.get().tableExists(dbName, tableName)) {
         client.get().dropTable(dbName, tableName);
-        HiveMetaStoreEventHelper.submitSuccessfulTableDrop(eventSubmitter, dbName, tableName);
+        String metastoreURI = this.clientPool.getHiveConf().get(HiveMetaStoreClientFactory.HIVE_METASTORE_TOKEN_SIGNATURE, "null");
+        String azkabanUrl = this.props.getProp(ConfigurationKeys.AZKABAN_EXECUTION_URL, "null");
```

Comment:
Please don't include azkaban specific stuff here. Instead, add it at the `RootMetricContext` level, in that case it will be an automatic tag everywhere.